service/create/security_group.go

```diff
@@ -71,30 +71,34 @@ func (s *Service) deleteSecurityGroup(input securityGroupInput) error {
 }
 
 // masterRules returns the rules for the masters security group.
-func (ri rulesInput) masterRules() []awsresources.Rule {
-	return []awsresources.Rule{
+func (ri rulesInput) masterRules() []awsresources.SecurityGroupRule {
+	return []awsresources.SecurityGroupRule{
 		{
 			Port:       ri.Cluster.Spec.Cluster.Kubernetes.API.SecurePort,
 			SourceCIDR: defaultCIDR,
 		},
 		{
 			Port:            ri.Cluster.Spec.Cluster.Etcd.Port,
-			SecurityGroupID: ri.OwnSecurityGroupID,
+			SecurityGroupID: ri.MastersSecurityGroupID,
```

Comment:
This rule should be removed. Its not needed and caused problems in my testing. The default SG is that all traffic is open within the SG. The etcd elb and the master node are in the same SG.
![screen shot 2017-05-11 at 2 23 59 pm](https://cloud.githubusercontent.com/assets/311527/25948940/0dfcc70e-3656-11e7-85e2-032753658450.png)